appendix/backup-and-restore/docker-compose.rst

```diff
@@ -0,0 +1,151 @@
+Backup & Restore (Docker)
+=========================
+
+This section shows some basics about the backup and restore process for a docker
+compose based deployment of Zammad. If you already use a different method or
+tool, it is fine to stick with it as long as it works. In such a case, consider
+using the :ref:`disable-backup-service scenario <additional-scenarios>`.
+
+If you're familiar with docker, the Quick Start section below includes the
+information you'll need. The :ref:`file-handling` section covers some examples
+about how to handle the backup files and to copy it into a docker volume to
+restore it.
+
+Quick Start
+-----------
+
+Backup
+^^^^^^
+
+By default, a backup is created at each start of the stack as well as at 3
+o'clock each night. The backup is stored in the volume of the
+**zammad-backup** container under ``/var/tmp/zammad``.
+
+Restore
+^^^^^^^
+
+.. warning::
```

Comment:
This should be no warning. Perhaps just add it to "Copy or move the backup files" below. People can either move all files (then the most recent ones will be used), or they can copy only a specific version of db and app file.